subsys/bluetooth/mesh/adv.h

```diff
@@ -13,8 +13,8 @@
 #define BT_MESH_ADV(buf) (*(struct bt_mesh_adv **)net_buf_user_data(buf))
 
 #define BT_MESH_ADV_SCAN_UNIT(_ms) ((_ms) * 8 / 5)
-#define BT_MESH_SCAN_INTERVAL_MS 30
-#define BT_MESH_SCAN_WINDOW_MS   30
+#define BT_MESH_SCAN_INTERVAL_MS 3000
+#define BT_MESH_SCAN_WINDOW_MS   3000
```

Comment:
Could you check if it works well with ZLL if the extended advertiser is enabled? And I doubt it works with the legacy advertiser. Perhaps this should be only enabled when the extended advertiser is enabled:
```suggestion
#if defined(CONFIG_BT_EXT_ADV)
#define BT_MESH_SCAN_INTERVAL_MS 3000
#define BT_MESH_SCAN_WINDOW_MS   3000
#else
#define BT_MESH_SCAN_INTERVAL_MS 30
#define BT_MESH_SCAN_WINDOW_MS   30
#endif
```
But please check with ZLL, otherwise it should be guarded with `CONFIG_BT_LL_SW_SPLIT`.

Reply:
it is correct, legacy advertiser is not happy with it. It is better increasing scan window only when ext. adv. is enabled.